Source/WebCore/loader/ImageLoader.cpp

```diff
@@ -429,8 +423,6 @@ void ImageLoader::notifyFinished(CachedResource& resource, const NetworkLoadMetr
     }
 
     m_imageComplete = true;
-    if (!hasPendingBeforeLoadEvent())
-        updateRenderer();
```

Comment:
The last I checked, this updateRenderer had effect.

Reply:
I think we need to do that in order to not break the test, or we need to debug what has changed to cause test failures and fix them.

Reply:
In a way, the only reason this code still exists is to call updateRenderer at the right time.